CheckDrive.Web/CheckDrive.Web/ViewModels/DashboardViewModel.cs

```diff
@@ -1,9 +1,135 @@
+                Status=DebtStatus.PartiallyPaid,
+            },
+            new Debt()
+            {
+                Id=3,
+                Driver="Shohruh Fozilov",
+                FuelAmount=20,
+                PaidAmount=20,
+                Petrol=PetrolType.Ai91,
+                Status=DebtStatus.Paid,
+            },
+            new Debt()
+            {
+                Id=4,
+                Driver="John Doe",
+                FuelAmount=50,
+                PaidAmount=10,
+                Petrol=PetrolType.Ai92,
+                Status=DebtStatus.PartiallyPaid,
+            },
+            new Debt()
+            {
+                Id=5,
+                Driver="Dilshod Ravshanov",
+                FuelAmount=40,
+                PaidAmount=0,
+                Petrol=PetrolType.Diesel,
+                Status=DebtStatus.Unpaid,
+            },
+            new Debt()
+            {
+                Id=6,
+                Driver="Sardor Jo'rayev",
+                FuelAmount=30,
+                PaidAmount=10,
+                Petrol=PetrolType.Ai92,
+                Status=DebtStatus.PartiallyPaid,
+            },
+            new Debt()
+            {
+                Id=7,
+                Driver="Jahongir Qobilov",
+                FuelAmount=40,
+                PaidAmount=40,
+                Petrol=PetrolType.Ai80,
+                Status=DebtStatus.Paid,
+            },
+            new Debt()
+            {
+                Id=8,
+                Driver="Rustam Ilhomov",
+                FuelAmount=10,
+                PaidAmount=0,
+                Petrol=PetrolType.Diesel,
+                Status=DebtStatus.Unpaid,
+            },
+            new Debt()
+            {
+                Id=9,
+                Driver="Javohir Javohirov",
+                FuelAmount=50,
+                PaidAmount=10,
+                Petrol=PetrolType.Ai95,
+                Status=DebtStatus.PartiallyPaid,
+            },
+            new Debt()
+            {
+                Id=10,
+                Driver="Sarvar Abduqodirov",
+                FuelAmount=70,
+                PaidAmount=0,
+                Petrol=PetrolType.Ai91,
+                Status=DebtStatus.Unpaid,
+            },
+        ];
+
+    public IEnumerable<PetrolCount> PetrolCounts =
+        [
+            new PetrolCount()
+            {
+                Type=PetrolType.Ai80,
+                Count=15
+            },
+            new PetrolCount()
+            {
+                Type=PetrolType.Ai91,
+                Count=10
+            },
+            new PetrolCount()
+            {
+                Type=PetrolType.Ai95,
+                Count=5
+            },
+            new PetrolCount()
+            {
+                Type=PetrolType.Ai92,
+                Count=2
+            },
+            new PetrolCount()
+            {
+                Type=PetrolType.Diesel,
+                Count=9
+            },
+
+        ];
```

Comment:
You dont give values to viewModels there . You only creaet viewModels and their properties. Values are gets in mock stores like this https://github.com/DiyorMarket/CheckDrive.Web/blob/integration/CheckDrive.Web/CheckDrive.Web/Stores/Technicians/MockTechnicianDataStore.cs